pyad/reverse_mode.py

```diff
@@ -146,3 +145,23 @@ def sqrt(x):
 
 def cbrt(x):
 	return _elementary_op(x, np.cbrt, lambda x: 1 / (3 * x**(2/3)))
+
+
+# Simple test suite explaining how Reverse works
+#Initialization
+x = Reverse(0.5)
+y = Reverse(4.2)
+z = Reverse(3)
+f = x * y**3 + sin(x) - logistic(z)
+
+#set df seed to be 1
+f.grad_value = 1.0
```

Comment:
Can we have a function that does the seeding? In general, setting attributes like this isn't very good from the abstraction point of view. So, it would be a better abstraction to the user if they wrote:

```
f.backward()
f_val = ...
x_val = ...
x_grad = ...
```
instead of
```
f.grad_value = 1.0
f_val = ...
x_val = ...
x_grad = ...
```